hugo/config/_default/menus.toml

```diff
@@ -34,6 +34,18 @@
     identifier = "sponsors"
     url = "/sponsors"
 
+[[main]]
+    title = "About"
+    weight = 7
+    identifier = "about"
+    url = "/about"
+
+[[main]]
+    title = "Contact"
+    weight = 8
+    identifier = "contact"
+    url = "/contact"

```

Comment:
Imo that should go into the footer instead of the main menu

Reply:
On many websites it is listed in the Menu and honestly I think no is really paying attention to footers (thats probably why so many companies put their legal stuff into that :stuck_out_tongue: ).
But it's up to you.

Reply:
We simply don't have the space in the menu. Plus most pages I have seen have the contact information in the footer :shrug: 
So I still think that's where we should put it.

Reply:
I would like to have it in the Menu, but having it in the footer is better than not having it at all.